pt-BR/mail/chrome/messenger/pbMessenger.dtd

```diff
@@ -138,223 +138,223 @@
+<!ENTITY spellAddToDictionary.label                  "Adicionar ao dicionário">
 <!ENTITY spellAddToDictionary.accesskey              "o">
-<!ENTITY spellNoSuggestions.label                    "(No Spelling Suggestions)">
-<!ENTITY spellCheckIgnoreWord.label                  "Ignore Word">
+<!ENTITY spellNoSuggestions.label                    "(Sem sugestões de ortografia)">
+<!ENTITY spellCheckIgnoreWord.label                  "Ignorar palavra">
 <!ENTITY spellCheckIgnoreWord.accesskey              "I">
-<!ENTITY spellCheckEnable.label                      "Check Spelling">
+<!ENTITY spellCheckEnable.label                      "Verificar ortografia">
 <!ENTITY spellCheckEnable.accesskey                  "S">
-<!ENTITY spellDictionaries.label                     "Languages">
+<!ENTITY spellDictionaries.label                     "Idiomas">
 <!ENTITY spellDictionaries.accesskey                 "L">
 
 <!-- indexer progress dialog -->
-<!ENTITY indexerProgress.title                       "Indexing Messages">
-<!ENTITY indexerProgressMessages.label               "Messages:">
-<!ENTITY indexerProgressFolders.label                "Folders:">
+<!ENTITY indexerProgress.title                       "Indexando mensagens">
+<!ENTITY indexerProgressMessages.label               "Mensagens:">
+<!ENTITY indexerProgressFolders.label                "Pastas:">
 <!ENTITY indexerProgress.status                      "Status:">
-<!ENTITY stopIndexingButton.label                    "Stop">
-<!ENTITY indexerProgressTitle.label                  "Indexing Messages...">
+<!ENTITY stopIndexingButton.label                    "Parar">
+<!ENTITY indexerProgressTitle.label                  "Indexando mensagens...">
 
 <!-- rich search bar -->
-<!ENTITY searchbar-defaultSearchText.label      "Search">
-<!ENTITY searchbar-caption.label                "Press Ctrl-ENTER for text preview">
-<!ENTITY searchbar-captionMac.label             "Press command-return for text preview">
+<!ENTITY searchbar-defaultSearchText.label      "Pesquisar">
+<!ENTITY searchbar-caption.label                "Pressione Ctrl-ENTER para visualizar o texto">
+<!ENTITY searchbar-captionMac.label             "Pressione command-return para visualizar o texto">
 
 <!-- advanced search panel -->
-<!ENTITY searchOptionsPanelTitle.label          "Search Options">
-<!ENTITY searchContainsField.label              "Contains">
-<!ENTITY searchFromField.label                  "From">
-<!ENTITY searchToField.label                    "To">
-<!ENTITY searchSubjectField.label               "Subject">
-<!ENTITY searchTopicField.label                 "Topic">
-<!ENTITY searchBeforeField.label                "Before">
-<!ENTITY searchAfterField.label                 "After">
+<!ENTITY searchOptionsPanelTitle.label          "Opções de pesquisa">
+<!ENTITY searchContainsField.label              "Contém">
+<!ENTITY searchFromField.label                  "De">
```

Comment:
CORRECTIONS NEEDED it should be +<!ENTITY searchFromField.label                  "Remetente">